src/components/GlobalPreferences/Notifications/NotificationsPreferences.js

```diff
@@ -58,23 +60,38 @@ export default function NotificationsPreferences({
                 mode="strong"
                 onClick={onLockSettings}
               >
-                Lock settings
+                <IconLock />
+                <span
+                  css={`
+                    width: ${12 * GU}px;
```

Comment:
Not a huge deal, but setting widths on `spans` is not very common. We most likely want to set a width on the `Button` instead?